### DISCUSSION TITLE
Netlify access amendments

### DISCUSSION THREAD

**Discussion by divya-mohan0209** [kubernetes/website#38019] [General]
Further to our conversation during Docs sprint at the Kubernetes contributor summit NA, this discussion has been opened for inputs from the wider community since there is a lot to understand reg. Netlify accesses

- One member from CNCF's Dev Advocate/Tech writer team to have access to K8s Netlify
--> Identifying the right amount of privilege to be assigned - collaborator/owner
--> Communicate to the wider community via an issue & seek feedback
--> Document how to go about doing it + when/how to remove the access

- Netlify access cleanup for emeritus co-chairs/tech leads
--> Formalise this as a part of the offboarding process + document it

- Netlify access cleanup for other non-Docs accounts
--> Flagging it to the correct points of contact & identifying the process around removals/cleanup

**Comment by divya-mohan0209** [comment]
Slack message: https://kubernetes.slack.com/archives/CA1MMM1HU/p1670378200284549

**Comment by divya-mohan0209** [comment]
Email sent to the DL: https://groups.google.com/g/kubernetes-sig-docs/c/2PT-ysHAric/m/OcUu9yr5AAAJ?utm_medium=email&utm_source=footer

**Comment by jimangel** [comment]
Big +1, if it's not too much scope creep, I think it would be great to outline how other projects are onboarded or gain access (via issue? DM? 🤷 ?)

Ref: https://kubernetes.slack.com/archives/CCTBSCVB9/p1667861269263849 (an approver for azure-api-provider requested access to their sites)

**Comment by divya-mohan0209** [comment]
Closing this discussion since Docs-related audit + documentation have been PR'ed in. For non-docs accounts, we will be pursuing the case via [this issue](https://github.com/kubernetes/community/issues/7311) on k/community.
